src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -135,8 +137,8 @@
 
```

Comment:
Are the lines above 124-136 needed? I think they are setting variables that aren't used and the border-block-end is using a variable that isn't set?